.NET/Microsoft.Recognizers.Text.Number/Parsers/BaseNumberParser.cs

```diff
@@ -26,6 +26,16 @@ public BaseNumberParser(INumberParserConfiguration config)
                                 GetKeyRegex(this.Config.CardinalNumberMap.Keys) + "|" +
                                 GetKeyRegex(this.Config.OrdinalNumberMap.Keys);
 
+            /*For Italian, we invert the order of Cardinal and Ordinal in singleIntFrac in order to correctly extract

```

Comment:
Nit: Same,

Reply:
Please add a `@TODO` code comment here to analyze remodeling the creation of this regex in the future; to call attention to it.